### PR TITLE
Improve code readability

### PR DESCRIPTION
**Pull request by miguelaferreira**
Also removed the comment because it seems obvious now.

### CODE REVIEW

File: templates/default/cg_foodcritic_tests.rb.erb [hunk 1]
```diff
@@ -39,6 +39,9 @@ def dependency_versions(ast)
   deps
 end
 
+def depends_on_specific_version(version)
```
Comment (svanharmelen) [inline templates/default/cg_foodcritic_tests.rb.erb]: In ruby it's custom to have a question-mark in the function name if it returns a bool.. .So this should be `def depends_on_specific_version?(version)` right?

**Comment by svanharmelen** [comment]
One small comment, good otherwise!

File: templates/default/cg_foodcritic_tests.rb.erb (hunk 1 above)
Comment (miguelaferreira) [inline templates/default/cg_foodcritic_tests.rb.erb]: indeed!

File: templates/default/cg_foodcritic_tests.rb.erb [hunk 2]
```diff
@@ -47,12 +50,8 @@ rule 'CG001', 'Ensure all cookbook dependencies depend on a specific version' do
     name = ast.xpath(%q{//command[ident/@value='name']})
 
     if name.to_s.match('<%= @matches %>')
-      # Get and loop through the hash with dependencies
-      # and check if they depend on a specific version
       dependency_versions(ast).each do |name,version|
-        unless version.to_s != '' && /^(?:= )?\d+\.\d+\.\d+$/ === version.xpath(%q{@value}).to_s
-          matches << name
-        end
+        matches << name unless depends_on_specific_version(version)
```
Comment (svanharmelen) [inline templates/default/cg_foodcritic_tests.rb.erb]: Uuh... Then we need the question-mark here as well of course :wink:

Comment (miguelaferreira) [inline templates/default/cg_foodcritic_tests.rb.erb]: You are right!!

Please let me do this the right way. I'm editing it on github. 

I will do a checkout, and re-do this PR. Sorry!

**Comment by svanharmelen** [comment]
LGTM!

**Pull request merged**